api/app/src/command/medical/document/document-download.ts

```diff
@@ -18,10 +20,10 @@ export const downloadDocument = async ({
 }): Promise<string> => {
   const { exists, contentType } = await doesObjExist({ fileName });
 
-  if (!exists) throw new Error("File does not exist");
+  if (!exists) throw new NotFoundError("File does not exist");
 
-  if (contentType !== "application/xml" && contentType !== "text/xml")
-    throw new Error("File is not a PDF or HTML");
+  if (conversionType && contentType !== "application/xml" && contentType !== "text/xml")
+    throw new BadRequestError("File is not a PDF or HTML");
```

Comment:
just noticed this - the message doesn't seem to be accurate.

would this be better? `Source file must be xml to convert to ${conversionType}, but it was ${contentType}`